forms/HtmlEditorField.php

```diff
@@ -323,8 +323,12 @@ public function MediaForm() {
 		$fileField->setAttribute('data-multiselect', true);
 		$columns = $fileField->getConfig()->getComponentByType('GridFieldDataColumns');
 		$columns->setDisplayFields(array(
-			'CMSThumbnail' => false,
-			'Name' => _t('File.Name'),
+			'StripThumbnail' => false,
+			'Title' => _t('File.Title'),
+			'Created' => singleton('File')->fieldLabel('Created'),
+		));
+		$columns->setFieldCasting(array(
+			'Created' => 'Date->Nice'
```

Comment:
Here's how it looks with date + time (`'Created' => 'SS_Datetime->Nice'`). I can't see an easy way to style the time differently than the date as they're not in separate html elements, but I don't think it looks too bad this way. Now that I've tried it out I don't think it looks too cluttered and I can see how it would be useful to some people so I'm in favour of including the time after all if you guys agree?

By the way I thought 'Uploaded' might be a better label than 'Created' for that field but I wasn't sure what implications changing that would have for i18n so I didn't go there.

![screen shot 2015-07-30 at 9 17 48 am](https://cloud.githubusercontent.com/assets/1079425/8988582/ea750c08-369b-11e5-8356-fe86f0f555b3.png)